torchaudio/datasets/cmuarctic.py

```diff
@@ -76,9 +76,15 @@ def load_cmuarctic_item(line: str,
 
 
 class CMUARCTIC(Dataset):
-    """
-    Create a Dataset for CMU_arctic. Each item is a tuple of the form:
```

Comment:
Do you really want to remove the information about the structure of these datasets?

Reply:
It's moved to `__getitem__` method.

Reply:
I think describing it in `__getitem__` has better locality, which is better for the developer experience. If something is changed on the implementation side, they only need to change the corresponding docstring and there are less chances that there are discrepancy between the docstring description and the actual implementation, compared to the case where the same information are repeated in class description.
Of course, we are having the current docathon to catch such discrepancies, so one can argue that it's okay to have such discrepancies, but it will confuse users on master branch still.